Client/WebFilters/AdblockResourcesMappings.swift

```diff
@@ -0,0 +1,159 @@
+        case ResourceNames.hi.rawValue: locale = Locales.hi.rawValue
+        case ResourceNames.fa.rawValue: locale = Locales.fa.rawValue
+        case ResourceNames.is.rawValue: locale = Locales.is.rawValue
+        case ResourceNames.he.rawValue: locale = Locales.he.rawValue
+        case ResourceNames.it.rawValue: locale = Locales.it.rawValue
+        case ResourceNames.ja.rawValue: locale = Locales.ja.rawValue
+        case ResourceNames.ko.rawValue: locale = Locales.ko.rawValue
+        case ResourceNames.lt.rawValue: locale = Locales.lt.rawValue
+        case ResourceNames.lv.rawValue: locale = Locales.lv.rawValue
+        case ResourceNames.nl.rawValue: locale = Locales.nl.rawValue
+        case ResourceNames.pl.rawValue: locale = Locales.pl.rawValue
+        case ResourceNames.ru.rawValue: locale = Locales.ru.rawValue
+        case ResourceNames.uk.rawValue: locale = Locales.uk.rawValue
+        case ResourceNames.be.rawValue: locale = Locales.be.rawValue
+        case ResourceNames.es.rawValue: locale = Locales.es.rawValue
+        case ResourceNames.sl.rawValue: locale = Locales.sl.rawValue
+        case ResourceNames.sv.rawValue: locale = Locales.sv.rawValue
+        case ResourceNames.tr.rawValue: locale = Locales.tr.rawValue
+        case ResourceNames.vi.rawValue: locale = Locales.vi.rawValue
+        default: locale = nil
+        }
+        
+        return locale
+    }
+}
+
+fileprivate enum ResourceNames: String {
+    case ar = "9FCEECEC-52B4-4487-8E57-8781E82C91D0-latest"
+    case bg = "FD176DD1-F9A0-4469-B43E-B1764893DD5C-latest"
+    case zh = "11F62B02-9D1F-4263-A7F8-77D2B55D4594-latest"
+    case cs = "7CCB6921-7FDA-4A9B-B70A-12DD0A8F08EA-latest"
+    case de = "E71426E7-E898-401C-A195-177945415F38-latest"
+    case da = "9EF6A21C-5014-4199-95A2-A82491274203-latest"
+    case et = "0783DBFD-B5E0-4982-9B4A-711BDDB925B7-latest"
+    case fi = "1C6D8556-3400-4358-B9AD-72689D7B2C46-latest"
+    case fr = "9852EFC4-99E4-4F2D-A915-9C3196C7A1DE-latest"
+    case el = "6C0F4C7F-969B-48A0-897A-14583015A587-latest"
+    case hu = "EDEEE15A-6FA9-4FAC-8CA8-3565508EAAC3-latest"
+    case id = "93123971-5AE6-47BA-93EA-BE1E4682E2B6-latest"
+    case hi = "4C07DB6B-6377-4347-836D-68702CF1494A-latest"
+    case fa = "C3C2F394-D7BB-4BC2-9793-E0F13B2B5971-latest"
+    case `is` = "48796273-E783-431E-B864-44D3DCEA66DC-latest"
+    case he = "85F65E06-D7DA-4144-B6A5-E1AA965D1E47-latest"
+    case it = "AB1A661D-E946-4F29-B47F-CA3885F6A9F7-latest"
+    case ja = "03F91310-9244-40FA-BCF6-DA31B832F34D-latest"
+    case ko = "1E6CF01B-AFC4-47D2-AE59-3E32A1ED094F-latest"
+    case lt = "4E8B1A63-DEBE-4B8B-AD78-3811C632B353-latest"
+    case lv = "15B64333-BAF9-4B77-ADC8-935433CD6F4C-latest"
+    case nl = "9D644676-4784-4982-B94D-C9AB19098D2A-latest"
+    case pl = "BF9234EB-4CB7-4CED-9FCB-F1FD31B0666C-latest"
+    case ru, uk, be = "80470EEC-970F-4F2C-BF6B-4810520C72E6-latest"
```

Comment:
This doesn't work, `ru` and `uk` string literals here are actually `"ru"`, and `"uk"`. Enum's raw values need to be unique and when defining `: String`, they default to themself (e.g. `case Foo` will have the string value `"Foo"` unless otherwise specified).

This is treated as three cases:
```
case ru
case uk
case be = "80470EEC-970F-4F2C-BF6B-4810520C72E6-latest"
```

I think in order to deal with this, we should actually completely remove `ResourceNames` enum, and instead make this a calculated property off of `Locales`.

```
fileprivate enum Locales: String { 
   case ...

    var resourceName: String {
        switch self {
        case .ar: return "9FCEECEC-52B4-4487-8E57-8781E82C91D0-latest"
        ....
        }
    }
}
```